src/bot/commands/helpers/checkElevatedRights.ts

```diff
@@ -0,0 +1,9 @@
+import Config from '@config/Config';
+import { Collection, Role } from 'discord.js';
+
+const userHasElevatedRole = (rolesCache: Collection<string, Role>) => {
```

Comment:
I think just passing the member here, instead of the roles, would better fit to the logic as well as the name of the function.

Reply:
I made this change :)